### ISSUE TITLE
Going over the guide changes since TBC

### ISSUE THREAD

**Issue by Sefriol** [cralor/TourGuideClassic#3]
Creating an issue of this, so it can be referenced in future commits.

The reason why commit history is actually valuable, is that there are number of changes done in WOTLK (which most of the guides are now based on), to systems that were still core in TBC.

There are for example notes for getting new spell ranks / pet training which were obsolete in WOTLK, I think we should restore those (or integrate some other Vanilla guide).

In addition, I think there were some changes to XP in both TBC and WOTLK that might make some of the guides not so accurate (especially on later levels). This is quite important

**Comment by cralor** [comment]
Yeah. A huge scrub (or just start from scratch) of the guides is going to be 100% required.

As the core functionality reaches maturity, I'd like to potentially reach out to some folks that have made guides for the recent launch. They may be interested in developing a solution for TourGuide as well.

We will want to take Tekkub's documentation on guide creation and additional information and clean it up for publication on this Wiki and/or perhaps a little GitHub Pages landing.

**Comment by Sefriol** [comment]
Going over the original change log is actually pretty good reference. Each change is documented pretty well.

**Comment by ignamax** [comment]
Hello, any ideas why game won't load the addon you're coding? 
thanks in advance

**Comment by cralor** [comment]
Ill package it and take a look

**Comment by Sefriol** [comment]
@ignamax is TourGuide visible in your addon list? If not, you must change the folder name to match addon name (i.e. not `TourGuideClassic`, but `TourGuide`).

That's the only thing I could think of atm. I have not had complete Addon loading problems.

IF this does not solve this, can you create a new issue with complete list of your loaded addons?

EDIT: Also, we do not have classic version on the Releases-page, since this is work in progress.

**Comment by Sefriol** [comment]
https://github.com/cralor/TourGuideClassic/tree/feature-darkshore

For human part I noticed, that we should bring back the parts that lead into Darkshore. Now it's kinda "just go Darkshore" without any lead up.

EDIT: Renamed because multiple zones involved.

**Comment by gitBurmecia** [comment]
Hinterlands in particular is littered with BC only quests, could likely get hold of a copy of joana's guide to import into this or use the guide from "AllyRoute" which is a leveling addon that has a decent route but its functionality is dog doo

**Comment by Sefriol** [comment]
If you have any links to particular guides that you find good (complete guides, zone guides etc), please do link them here. Easier for us to go them through.

**Comment by cralor** [comment]
Yup -- this is the next major step.

I'd like to reach out to some guide authors and ask for permission and/or
their interest in porting their guides themselves.
